### ISSUE TITLE
Groundhog Day - who's editing what?

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#110]

**Comment by cgrandin** [comment]
I'm automating prior values (as seen in the control file) and types in the text right now.

**Comment by allan-hicks** [comment]
I'll look through the one-pager and executive summary, but not until this
afternoon.

**Comment by cgrandin** [comment]
I will also go through as much of the doc as possible, finding any hard wired numbers, and try to generalize them if possible.

**Comment by andrew-edwards** [comment]
tables.reference.points.r - making row names of F_40% part in Table 13 consistent with those in Table f (and the identical Table 14)..

**Comment by minnowslayer** [comment]
i'm at this intermittently today but right now am identifying a couple
spots where years and other quantities are hard-wired in the executive
summary

**Comment by minnowslayer** [comment]
but all done that now

**Comment by allan-hicks** [comment]
Is there a way to avoid us changing the same things? How about if we wait
for Chris and Andy to go through looking for hard-wired quantities. Then we
can have a closer look later to make sure it is correct?

If so, Chris and Andy, can you let us know what sections are finished and
ready for further review?

**Comment by andrew-edwards** [comment]
If we agree to say here what section we are currently editing, then others can check. Yes, we want to avoid duplicating the work. We can merge frequently. I'm still doing what I said I was above (tables.reference.points.r).

**Comment by minnowslayer** [comment]
no problem.

**Comment by allanhicks** [comment]
I'm not editing anything right now, but will start after lunch.

**Comment by andrew-edwards** [comment]
Done with aforementioned table edit.
Will do Exec Summary - Nathan flagged some stuff which I can do now I have new models all working fine.

**Comment by aaronmberger** [comment]
I had done some updating last night on the assessment section, and noted in
the assessment.rnw which ones were '2016 done'; however a good check over
would be prudent either by myself or another.  In most spots, I just
changed it to the new value.  Some of those would be difficult to automate,
but some would be fairly simple.

On Thu, Feb 11, 2016 at 10:37 AM, andrew-edwards <notifications@github.com>
wrote:

> Done with aforementioned table edit.
> Will do Exec Summary - Nathan flagged some stuff which I can do now I have
> new models all working fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cgrandin/hake-assessment/issues/110#issuecomment-183001313>
> .
>



-- 

*Aaron M. Berger, Ph.D. Research Mathematical Statistician*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2032 SE OSU Dr. Bldg. 955*
*Newport, OR 97365-5275*



*aaron.berger@noaa.gov <aaron.berger@noaa.gov>541.867.0562*

**Comment by andrew-edwards** [comment]
Done with Exec Summary. 
Will do One-page Summary after lunch as I flagged three things earlier than I can easily verify now I have results.
New issue - hungry.

**Comment by allanhicks** [comment]
I'll start going through the Exec Summary, if that is ok with everyone.

**Comment by cgrandin** [comment]
Yes, fine by me

**Comment by andrew-edwards** [comment]
I'm going to print and read Exec Summary, as never really did that last time. Oh, just have two minor punctuation edits for data section first.

**Comment by allan-hicks** [comment]
I cna commit real quick if you would like to see the changes I've made

**Comment by andrew-edwards** [comment]
Yes, I'm actually just doing the bit in Assessment section that Aaron flagged - I'd just done the same paragraph in Exec Summary.

**Comment by cgrandin** [comment]
I' going through sensitivity and retrospectives sections.

**Comment by cgrandin** [comment]
I'm heading out to pick up my kids and go home (Nadine's first day back at work after mat leave) but I will check in in an hour or so. Cheers

**Comment by aaronmberger** [comment]
I also have to head out in a bit and drive back to Corvallis... but will
check in a bit later...

On Thu, Feb 11, 2016 at 3:05 PM, cgrandin <notifications@github.com> wrote:

> I'm heading out to pick up my kids and go home (Nadine's first day back at
> work after mat leave) but I will check in in an hour or so. Cheers
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cgrandin/hake-assessment/issues/110#issuecomment-183094342>
> .
>



-- 

*Aaron M. Berger, Ph.D. Research Mathematical Statistician*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2032 SE OSU Dr. Bldg. 955*
*Newport, OR 97365-5275*



*aaron.berger@noaa.gov <aaron.berger@noaa.gov>541.867.0562*

**Comment by allanhicks** [comment]
Sounds good. We are in a good place and should have a document to
distribute tomorrow before noon.
Thanks everyone.

**Comment by andrew-edwards** [comment]
Sounds good. I'm off soon and will get the Exec Summary read through (something I ran out of time to do on Monday, so that'll be good).
Thought I was up-to-date, but just seen a bunch more commits. I'll merge tomorrow.
160-page document updated in essentially a day (after you guys spent a couple of days doing model runs). Nice.

**Comment by aaronmberger** [comment]
Agreed.  All that hard work with set-up I thought would really pay off next
year, but seeing major benefits already!

On Thu, Feb 11, 2016 at 3:44 PM, andrew-edwards <notifications@github.com>
wrote:

> Sounds good. I'm off soon and will get the Exec Summary read through
> (something I ran out of time to do on Monday, so that'll be good).
> Thought I was up-to-date, but just seen a bunch more commits. I'll merge
> tomorrow.
> 160-page document updated in essentially a day (after you guys spent a
> couple of days doing model runs). Nice.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cgrandin/hake-assessment/issues/110#issuecomment-183111930>
> .
>



-- 

*Aaron M. Berger, Ph.D. Research Mathematical Statistician*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2032 SE OSU Dr. Bldg. 955*
*Newport, OR 97365-5275*



*aaron.berger@noaa.gov <aaron.berger@noaa.gov>541.867.0562*

**Comment by cgrandin** [comment]
Looks like everything is ok! I'll get to work in the morning to do a final check. Cheers guys!
